README.md

```diff
@@ -105,6 +106,7 @@ File Name | Dash Docset Keys | Docset Setting
 vagrantfile | vagrant | dash.fileNameToDocsetMap["vagrantfile"]
 gruntfile.js | grunt | dash.fileNameToDocsetMap["gruntfile.js"]
 gulpfile.js | gulp | dash.fileNameToDocsetMap["gulpfile.js"]
+*.ino | arduino | dash.fileNameToDocsetMap["*.ino"]
```

Comment:
this shouldn't be in this PR anymore, right? have you updated and pulled from the latest master?

Reply:
Sorry, I thought I did pull from latest master, but it looks like it did not work, so I hard resetted my fork and now it should be good :)! Sorry for the inconvenience.